modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotMessageCounter.java

```diff
@@ -140,12 +124,11 @@ public void recordMetadataCountInSlot(Collection<AndesMessage> messageList) {
      *
      * @param metadata AndesMessageMetadata
      */
-    public void recordMetadataCountInSlot(AndesMessageMetadata metadata) {
+    private void recordMetadataCountInSlot(AndesMessageMetadata metadata) {
```

Comment:
shoudn't we rename this to something like recordMessageCountInSlot or something. given that metadata : message is 1:1 relationship